fixtures/fovitec/600xb.json

```
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "600XB",
+  "categories": ["Other"],
+  "meta": {
+    "authors": ["Aninymous"],
+    "createDate": "2021-08-03",
+    "lastModifyDate": "2021-08-03"
+  },
+  "links": {
+    "productPage": [
+      "https://www.fovitec.com/products/600xb-bi-color-led-panel-with-dmx"
+    ]
+  },
+  "availableChannels": {
+    "Intensity": {
+      "capability": {
+        "type": "Intensity"
```

```suggestion
        "type": "Intensity",
        "brightnessStart": "0lm",
        "brightnessEnd": "1420.8lm"
```
Based on Beamangle 45° and 2,636 LUX at 1m. and Dimming Capability | 0-100% continuous